File: plugins/kco-mobile-sdk/cordova/KlarnaCheckoutPlugin.java
package com.klarna.checkout;

import android.app.Activity;
import android.webkit.WebView;
import org.apache.cordova.CordovaPlugin;

public class KlarnaCheckoutPlugin extends CordovaPlugin {

    @Override
    public void pluginInitialize() {

        final Activity activity = this.cordova.getActivity();
        final WebView webView = (WebView) this.webView.getView();
        final KlarnaCheckout checkout = new KlarnaCheckout(activity);
        checkout.setWebView(webView);
    }

}

File: platforms/ios/CordovaSampleApp/Plugins/kco-mobile-sdk/KlarnaCheckoutPlugin.m
#import "KlarnaCheckoutPlugin.h"
#import <Cordova/CDVPlugin.h>
#import "KlarnaCheckout/KlarnaCheckout.h"

@interface KlarnaCheckoutPlugin ()

@property (strong) KCOKlarnaCheckout *checkout;

@end


@implementation KlarnaCheckoutPlugin

-(void)pluginInitialize {
    self.checkout = [[KCOKlarnaCheckout alloc] initWithViewController:self.viewController];
    [self.checkout notifyViewDidLoad];
    [self.checkout setWebView:self.webView];
}

-(void)dispose{
    [self.checkout destroy];
}

@end